prow/oss/run-k8s-binary.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# This script is for running arbitrary Go binary from k8s/test-infra repo,
+# running this script when this script cannot run docker container, for example
```

Comment:
Can you reword `running this script when this script cannot run docker container`? If I read this right it says that I should run this script when this script cannot run a docker container, but I don't think that's what you meant.

EDIT: Oh I think you meant to say `when the binary cannot run inside a Docker container`.

Here's my take at rewording this comment:

```
# This script is for running arbitrary Go binaries from the k8s/test-infra repo.
# Running this script when the binary cannot run inside a Docker container (for example,
# when the binary needs to authenticate to GCP).
# This script installs the binary directly from the k8s/test-infra repo instead of adding
# k8s.io/test-infra as a Go module inside the current directory. This is because we've seen
# dependency problems when doing so in other repos.
```

WDYT?

Reply:
looks nice, I'll steal it